### PR TITLE
feat: APP-342 user profile certificate multiple denom

### PR DESCRIPTION
**Pull request by r41ph**
## Description

https://regennetwork.atlassian.net/browse/APP-341
https://regennetwork.atlassian.net/browse/APP-342

This PR also removes the Retirement reason heading from the retirement certificate if the user hasn't added a reason.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

https://deploy-preview-2533--regen-marketplace.netlify.app/
1. Verify that the Retirement Certificates table shows multiple batch IDs when in retirements from more that one batch - https://deploy-preview-2533--regen-marketplace.netlify.app/dashboard/portfolio
2. Verify the Retirement certificates shows the Credit batch/es at the bottom of the certificate
3. Verify that certificates don't show the Retirement reason heading if the user hasn't added a reason when buying credits.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by r41ph** [comment]
@erikalogie see testing instructions

**Comment by r41ph** [comment]
@erikalogie I have updated the credits batches list with the new design - https://deploy-preview-2533--regen-marketplace.netlify.app/certificate/WyJyZXRpcmVtZW50cyIsMSwxNzU1ODgwMywwLDBd

**Comment by r41ph** [comment]
@erikalogie as discussed I've added support for all the dropdowns to work together in each row and fixed the alignment issues in Brave, please have a look

File: web-marketplace/src/components/organisms/RetirementCertificatesTable/RetirementCertificatesTable.tsx [hunk 1]
```diff
@@ -60,6 +62,31 @@ export const RetirementCertificatesTable: React.FC<
...
+        <Link href={`/credit-batches/${batchId}`}>{batchId}</Link>
+      </WithLoader>
+    ));
+  };
+
+  const getBatchDate = (dates: string[]) => {
+    return dates.map(date => (
+      <GreyText className="whitespace-nowrap">{formatDate(date)}</GreyText>
+    ));
+  };
```
[the remark this answers is not in the document]
Comment (r41ph) [reply web-marketplace/src/components/organisms/RetirementCertificatesTable/RetirementCertificatesTable.tsx:81]: I've moved them to a _utils_ file

**Review by blushi — APPROVED** [review]
LGTM

**Pull request merged**